src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You should link a CSS file instead of SCSS. The browser doesn't understand SCSS syntax. You need to compile your SCSS to CSS first and then link the CSS file.